prow/cmd/pipeline/main_test.go

```diff
@@ -29,18 +29,25 @@ func TestOptions(t *testing.T) {
 		expected *options
 		err      bool
 	}{{
-		name:     "defaults work",
+		name:     "defaults doesn't work (--config argument is mandatory)",
```

Comment:
Please fix this so it provides the minimal amount necessary to work (aka add --config)

Reply:
I enhanced the validation message and the test name with a more explicit message about what should be done to make it work.